config/routes.rb

```diff
@@ -12,13 +12,16 @@
   end
 
   resources :message
+  post 'message/new' => 'message#create', :as=>"create_message"
 
-  resources :profile, only: [:show] do
+  resources :profile, only: [:show, :create] do
```

Comment:
dont think u need the create method here anymore since u alreayd create a new route for create_message